lib/rides/widgets/search_suggestion_filter.dart

```diff
@@ -419,14 +419,11 @@ class SearchSuggestionFilter {
           (Ride ride) {
             Profile driver = ride.drive!.driver!;
             AggregateReview driverReview = driver.getAggregateReview();
-            bool ratingSatisfied = driver.getAggregateReview().rating >= _minRating &&
-                (driverReview.comfortRating >= _minComfortRating ||
-                    _minComfortRating == 1 && !driverReview.isComfortSet) &&
-                (driverReview.safetyRating >= _minSafetyRating || _minSafetyRating == 1 && !driverReview.isSafetySet) &&
-                (driverReview.reliabilityRating >= _minReliabilityRating ||
-                    _minReliabilityRating == 1 && !driverReview.isReliabilitySet) &&
-                (driverReview.hospitalityRating >= _minHospitalityRating ||
-                    _minHospitalityRating == 1 && !driverReview.isHospitalitySet);
+            bool ratingSatisfied = (driverReview.rating >= _minRating || !driverReview.isRatingSet) &&
```

Comment:
I would switch these conditions. This current way the code relies on the internal behaviour that ("rating set" == "rating == 0"). But that part might change in the future.